internal/server/stat/golden/get_stat_all/result.json

```diff
@@ -1343,16 +1343,6 @@
                 "1997": 70698,
                 "1998": 71479,
                 "1999": 71470,
-                "2000": 71241,
-                "2001": 70919,
-                "2002": 70245,
-                "2003": 69662,
-                "2004": 69167,
-                "2005": 69479,
-                "2006": 69744,
-                "2007": 70352,
-                "2008": 71927,
-                "2009": 73066,
```

Comment:
USCensusPEP_Annual_Population does have data for country/USA
https://screenshot.googleplex.com/75TTP4dRXervn5Y

Reply:
These deletion are gone now.